tests/MongoDB.EntityFrameworkCore.FunctionalTests/Query/OwnedEntityTests.cs

```diff
@@ -41,6 +41,103 @@ public void OwnedEntity_nested_one_level_materializes_single()
         Assert.Equal(Location1.longitude, actual.location.longitude);
     }
 
+    [Fact]
+    public void OwnedEntity_nested_one_level_where_null()
```

Comment:
This and next test have confusing names. `where_null`, but we actually here we filter by != null. When next test named as `where_not_null` and there we filtering by == null.

Reply:
Nice catch, fix pushed.